proto/fungible/query.proto

```diff
@@ -63,3 +73,21 @@ message QueryGetSystemContractRequest {}
 message QueryGetSystemContractResponse {
   SystemContract SystemContract = 1 [(gogoproto.nullable) = false];
 }
+
+message QueryGetGasStabilityPoolAddress {
+  int64 chain_id = 1;
```

Comment:
ref. other comment
if only one pool address for all chains, then this query can be simplied without chain_id

Reply:
ecbff622